Cargo.lock

```diff
@@ -3090,7 +3100,7 @@ dependencies = [
 name = "nu-table"
 version = "0.86.1"
 dependencies = [
- "fancy-regex",
+ "fancy-regex 0.11.0",
```

Comment:
Super weird that dependabot doesn't bump the version in `nu-table`

Reply:
Oof, maybe because it is neither an explicit workspace member or a direct dependency of `nu`?

https://github.com/nushell/nushell/blob/a806717f35e486fbc1c203b3421e37021f2431b1/Cargo.toml#L25-L70